packages/aztec.js/src/proof/proofCatalog.js

```diff
@@ -0,0 +1,48 @@
+const BurnProof66305 = require('./epoch0/BURN/burn');
+const DividendProof66561 = require('./epoch0/UTILITY/dividend');
+const JoinSplitProof65793 = require('./epoch0/BALANCED/joinSplit');
+const MintProof66049 = require('./epoch0/MINT/mint');
+const PrivateRangeProof66562 = require('./epoch0/UTILITY/privateRange');
+const PublicRangeProof66563 = require('./epoch0/UTILITY/publicRange');
+const SwapProof65794 = require('./epoch0/BALANCED/swap');
+
+const proofCatalog = {};
+
+/*
+ * Default epoch number for which proof construction methods are exported
+ */
+proofCatalog.defaultEpochNum = 1;
```

Comment:
The way this is currently structured, it assumes that we will never add an epoch on one proof without also adding proofs for all the existing proof types for that new epoch. Is that true?

Reply:
As discussed earlier, I think `aztec.js` proof epochs can be added to - we don't need to add all new proofs to an epoch at the same time. Will leave as is for now